common/resources.yaml

```diff
@@ -0,0 +1,67 @@
+groups:
+ - data_type_def: ExternalResources
```

Comment:
I thought we were going to use a DynamicTable for this

Reply:
No, it's a set of tables that we don't want to be customizable.